scripts/dev_scripts/integration_tests_docker.sh

```diff
@@ -58,7 +58,7 @@ python $COMPARE_POLICIES $POLICY_RESULT $POLICY_EXPECTED || log_fail
 # Clean up and remove the virtual environment.
 rm -rf "$VIRTUAL_ENV_PATH"
 
-python3 ./tests/integration/run.py run \
+python ./tests/integration/run.py run \
```

Comment:
I think we would need to change the same for `integration_tests.sh` too.

Reply:
Not sure how I missed that :-) 
See commit [ba87342](https://github.com/oracle/macaron/pull/783/commits/ba87342e2dfc5bffd5011f879f08ade784f6175d)